packages/url_launcher/lib/url_launcher.dart

```diff
@@ -20,9 +20,10 @@ const MethodChannel _channel = MethodChannel('plugins.flutter.io/url_launcher');
 /// [forceSafariVC] is only used in iOS. If unset, the launcher opens web URLs
 /// in the safari VC, anything else is opened using the default handler on the
```

Comment:
While you're here, consider replacing `safari VC` with `Safari view controller`.